#! /bin/sh -e
## 07_use_libtool.dpatch by Martin Loschwitz <madkiss@madkiss.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Enable optimization for all arches

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
    -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1;;
esac

exit 0

diff -ruN qt-x11-free-3.3.2.old/src/qt.pro qt-x11-free-3.3.2/src/qt.pro
--- qt-x11-free-3.3.2.old/src/qt.pro	2004-03-01 11:20:58.000000000 +0100
+++ qt-x11-free-3.3.2/src/qt.pro	2004-05-14 16:22:26.000000000 +0200
@@ -169,7 +169,7 @@
 !staticlib:PRL_EXPORT_DEFINES += QT_SHARED
 
 unix {
-   CONFIG     += create_libtool create_pc
+   CONFIG     += create_libtool
    QMAKE_PKGCONFIG_LIBDIR = $$target.path
    QMAKE_PKGCONFIG_INCDIR = $$headers.path
 }
